.gitignore

```diff
@@ -1,3 +1,28 @@
+
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+bin
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Kubernetes Generated files - skip generated files, except for vendored files
+
+!vendor/**/zz_generated.*
+
+# editor and IDE paraphernalia
+.idea
+*.swp
```

Comment:
maybe we can add `.vscode` as well.

Reply:
@kuritka it's actually there https://github.com/AbsaOSS/k8gb/blob/6d05dc68d0c52d70b45e2971e57294b69270bc6f/.gitignore#L100 
.gitignore is a bit messy given the merge with upstream, i just sequentially combined them as of now